modules/340-monitoring-kubernetes/hooks/choose_nodes_for_ebpf_exporter.go

```diff
@@ -96,6 +136,13 @@ func labelNodes(input *go_hook.HookInput) error {
 				return nil, err
 			}
 
+			// skip nodes with non-managed kernel, can't rely on the correct kernel and kernel headers being present
+			if has, ok := ngIsManagedKernelSet[nodeEligibility.NodeGroupName]; !ok || !has {
```

Comment:
Can't we run ebpf-exporter even on nodes with non-managed kernel?

Reply:
Removed the hook and the label.